### PR TITLE
Use com.android.support.test instead of com.androidx.test

### LINKED ISSUE #20748: Miss a '.support'
https://github.com/tensorflow/tensorflow/blob/365d2fc4d62540b2c6524500a7a58e7edab0dfa9/tensorflow/contrib/lite/java/demo/app/build.gradle#L47

It should be com.android.support.test instead of com.androidx.test. Otherwise, gradle will report an error

### PR DESCRIPTION
**Pull request by jmannanc**
This PR should fix https://github.com/tensorflow/tensorflow/issues/20748 and https://github.com/tensorflow/tensorflow/issues/20828

### CODE REVIEW

File: tensorflow/contrib/lite/java/demo/app/build.gradle [hunk 1]
```diff
@@ -44,7 +44,7 @@ repositories {
 
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
-    androidTestCompile('com.androidx.test.espresso:espresso-core:2.2.2', {
+    androidTestCompile('com.android.support.test.espresso:espresso-core:2.2.2', {
```
Comment (andrehentz) [inline tensorflow/contrib/lite/java/demo/app/build.gradle]: Could you try: 
   `￼androidx.test.espresso:espresso-core:3.1.0-alpha3`
￼instead?

Comment (SneakyFish5) [reply tensorflow/contrib/lite/java/demo/app/build.gradle]: Done

**Comment by SneakyFish5** [comment]
Was this fixed with https://github.com/tensorflow/tensorflow/commit/1d38580d6db3dc48b916453c2ce6cce691a00fe6? If so I'll just close this.

**Comment by SneakyFish5** [comment]
Yea, I'm all good thanks. Closing this now.

**Pull request closed without merging**